hnn_core/gui/_viz_manager.py

```diff
@@ -0,0 +1,575 @@
+"""HNN GUI visualization management tool"""
+
+# Authors: Huzi Cheng <hzcheng15@icloud.com>
+
+import copy
+import io
+from functools import partial
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from IPython.display import display
+from ipywidgets import (Box, Button, Dropdown, FloatText, HBox, Label, Layout,
+                        Output, Tab, VBox, link)
+
+from hnn_core.gui._logging import logger
+from hnn_core.viz import plot_dipole
+
+
+_dpi = 96
```

Comment:
where did you get this number? is this going to be the same across computers? Maybe you should [set_dpi](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.set_dpi) to ensure this works across screen resolutions?

Reply:
dpi now could be set from `HNNGUI`